yarn.lock

```diff
@@ -0,0 +1,7333 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
dammit, that was an accident. Can we just add it to the `.gitignore` file or will that screw up the deploy process?